eclass/python-utils-r1.eclass

```diff
@@ -1336,15 +1336,8 @@ epytest() {
 	_python_check_EPYTHON
 	_python_check_occluded_packages
 
-	local color
-	case ${NOCOLOR} in
-		true|yes)
-			color=no
-			;;
-		*)
-			color=yes
-			;;
-	esac
+	local color=yes
+	[[ ${NO_COLOR} ]] && color=no
```

Comment:
The upstream pytest behavior is even more fun here. It checks if it is in os.environ, which means even if it is set to the null string it would still disable colors.

So I think what you really want here is `[[ -v NO_COLOR ]]`.